components/script/dom/htmlinputelement.rs

```diff
@@ -731,6 +741,80 @@ impl HTMLInputElement {
         let el = self.upcast::<Element>();
         el.set_placeholder_shown_state(has_placeholder && !has_value);
     }
+
+    // https://html.spec.whatwg.org/multipage/#file-upload-state-(type=file)
+    // Select files by invoking UI or by passed in argument
+    fn select_files(&self, opt_test_paths: Option<Vec<DOMString>>) {
```

Comment:
This isn't process safe -- we should assert that the pref is enabled if opt_test_paths is nonempty here.